source/manual/schemas.html.md

```diff
@@ -0,0 +1,41 @@
+# Schemas used on GOV.UK
+Here are a list of schema's being used
+
+### [HowTo][how-to]
+
+[how-to]: https://schema.org/HowTo
```

Comment:
It would be nice to bundle all the link definitions together, either at the end of each section (preferred) or at the end of the document. This way, they don't break up the flow of a section.